.github/workflows/ci.yml

```diff
@@ -128,9 +132,39 @@ jobs:
 #          USE_FOUNDRY_BUNDLE: 1
           DATABASE_URL: sqlite:///%kernel.cache_dir%/app.db
 
+      - name: 'Test: MySQL & Mongo'
```

Comment:
maybe it's time to use a matrix for all test cases?

Reply:
Let's save for a future PR.